modules/code-builder/src/brick/design0/BrickExpression.ts

```diff
@@ -51,6 +51,34 @@ export default class BrickExpression extends BrickModelExpression {
     }
 
     public get SVG(): string {
-        return '';
+        return `<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Was confused about this part, can you please confirm once @meganindya ?

Reply:
Don't need it I think.

Reply:
I was talking about the actual SVG part; if that's the correct one!